src/tests/multihost/alltests/pytest.ini

```diff
@@ -22,6 +22,7 @@ markers =
     passwordcheck: Tests related to password check while updating
         password of user
     proxy: Tests related to sssd-proxy
+    proxy_provider : Tests related to Proxy Provider
```

Comment:
Line 24 is used as a marker, so we dont need 'proxy_provider' marker

Reply:
Done